docs/variables/countdown.md

```diff
@@ -4,33 +4,51 @@ id: countdown
 
 # $(countdown)
 
-Countdown tells you the amount of time until a certain date has passed.
+Returns the time that is left until a certain date has passed. Shares this behaviour with `$(countup)`.
 
-For example, `$(countdown 2022-01-01 00:00:00)` will make the bot calculate the time until 1st Jan 2022.
+#### Parameters
 
-```
-2 months, 10 days and 2 hours
-```
+This variable takes ***one*** *required* parameter that is a date string to count down to.
 
-To use a specific timezone you can add the UTC offset to the timestamp.
+* **Accepted Date Formats**
+  * `2030-01-01`
+  * `Jan 01 2030`
+  * `01 Jan 2030`
+  * `Jan 1st 2030`
+  * `1st Jan 2030`
```

Comment:
FWIW this supports way more, this is the library fossabot uses: https://github.com/araddon/dateparse

Reply:
I'll add a link to it.